src/main/java/com/iyzipay/model/ApiTest.java

```diff
@@ -1,12 +1,11 @@
 package com.iyzipay.model;
 
-import com.iyzipay.HttpClient;
 import com.iyzipay.IyzipayResource;
 import com.iyzipay.Options;
 
 public class ApiTest extends IyzipayResource {
 
     public static IyzipayResource retrieve(Options options) {
-        return HttpClient.create().get(options.getBaseUrl() + "/payment/test", IyzipayResource.class);
+        return options.getHttpClient().get(options.getBaseUrl() + "/payment/test", IyzipayResource.class);
```

Comment:
It is a bad practice to get HttpClient from options. Options class is just a Value Object carries api specific informations.